config/initializers/rollbar.rb

```diff
@@ -0,0 +1,52 @@
+require 'rollbar/rails'
+Rollbar.configure do |config|
+  # Without configuration, Rollbar is enabled in all environments.
+  # To disable in specific environments, set config.enabled=false.
+  
+  config.access_token = ENV['ROLLBAR_ACCESS_TOKEN']
+
+  # Here we'll disable in 'test':
+  if Rails.env.test?
```

Comment:
[rubocop] Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.